### ISSUE TITLE
Meter metrics not working anymore

### ISSUE THREAD

**Issue by bvis** [trocotronic/edistribucion#7]
Since some days ago the system does not obtain any meter when doing the call to the `consultarContador=1` endpoint in the e-distribution API:

```
root@082c6f328eba:/edistribucion# ./example.py
[12/08/2021 23:39:33] [INFO] Loging
[12/08/2021 23:39:33] [INFO] Preparing command: other.WP_ContadorICP_CTRL.getCUPSReconectarICP=1
[12/08/2021 23:39:34] [INFO] Sending POST request to https://zonaprivada.edistribucion.com/areaprivada/s/sfsites/aura?other.WP_ContadorICP_CTRL.getCUPSReconectarICP=1
[12/08/2021 23:39:34] [INFO] Response with code: 200
Cups:  XXXXXXXXXXXXX
[12/08/2021 23:39:34] [INFO] Preparing command: other.WP_ContadorICP_CTRL.consultarContador=1
[12/08/2021 23:39:34] [INFO] Sending POST request to https://zonaprivada.edistribucion.com/areaprivada/s/sfsites/aura?other.WP_ContadorICP_CTRL.consultarContador=1
[12/08/2021 23:39:34] [INFO] Response with code: 200
Meter:  None
```
As you may see in the logs the call returns a `200 OK` code but instead of obtain the usual data it gets an empty value. Could it be related to a change in the API? Do you have any documentation about how the API works?

I could dedicate some time, not so much, and provide a PR.

**Comment by trocotronic** [comment]
Should be fixed in 5e84d95.

**Comment by bvis** [comment]
Right! Fast as hell! Thanks
